packages/react-composites/tests/app/chat/LiveTestApp.tsx

```diff
@@ -62,7 +62,9 @@ export const LiveTestApp = (): JSX.Element => {
           fileUploads[0].notifyUploadCompleted({
             name: file.name,
             extension: file.extension,
-            url: file.url
+            url: file.url,
+            attachmentType: 'fileSharing',
```

Comment:
if I recall correctly, we were going with 'file' as the attachment type

Reply:
No this isn't related to the file attachment type that will be coming down from ChatSDK.